wordpress-mobile/WordPress-iOS
# All Time Widget - Tracks and deeplink

This PR adds tracks to the all time widget, as well as deep linking the tap to the 'Insights' page of the selected site.
It also updates the release notes

Fixes #NA

To test:

- build/run on iOS 14
- install one or more All Time widgets, and select sites of your preference on them
- make sure that tapping on a widget opens the 'Insights' page for the same site in the app
- In mission control, select the tracks live view tool (Metrics -> Tracks -> Live view), and input wpios_alltime_home_extension_widget_updated as event name
- Wait for at least five minutes (sometimes it takes 8/9 minutes) and see the incoming tracks in the log
- Expand the tracks and verify that the info medium_widgets, small_widgets and total_widgets are consistent with that you installed

Known issues: 
- two widgets of the same size and the same site, seem to be counted as one.
- some events may appear more than once in the live view

PR submission checklist:
